ocp_cleanup.sh

```diff
@@ -4,8 +4,10 @@ set -e
 
 source ocp_install_env.sh
 
-$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster || true
-
-rm -rf ocp
-
-sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf
+if $GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster; then
```

Comment:
```suggestion
if "$GOPATH/src/github.com/openshift/installer/bin/openshift-install" --log-level=debug --dir ocp destroy cluster; then
```

Reply:
I get quoting variables for string comparison in a test, but what's the logic behind quoting a command like this?

Reply:
GOPATH may contain spaces can't it?

Here's an example of the issue:

```
~ 🎩︎ echo $GOPATH
/home/celebdor/foo bar
~ 🎩︎ if $GOPATH/boo; then
echo "true"
else
echo "false"
fi
cd -- /home/celebdor/foo bar/boo
bash: cd: too many arguments
false
~ 🎩︎ if "$GOPATH/boo"; then
echo "true"
else
echo "false"
fi
true
```

Reply:
Hmm okay - spaces in paths are an evil Windows thing IMHO so I don't get why anyone would ever do that on Linux - a bunch of stuff will break if the space isn't escaped I expect.

Anyway, thanks for clarifying!